source/opt/ir_loader.cpp

```diff
@@ -35,12 +35,14 @@ IrLoader::IrLoader(const MessageConsumer& consumer, Module* m)
       module_(m),
       source_("<instruction>"),
       inst_index_(0),
+      last_line_inst_(nullptr),
```

Comment:
nit: unique_ptrs get `nullptr` by default, don't they? 

https://en.cppreference.com/w/cpp/memory/unique_ptr/unique_ptr
> Value-initializes the stored pointer and the stored deleter.

Value initialization: https://en.cppreference.com/w/cpp/language/value_initialization
> 4) otherwise, the object is zero-initialized.

Zero initialization: https://en.cppreference.com/w/cpp/language/zero_initialization
> If T is a scalar type, the object's initial value is the integral constant zero explicitly converted to T.

Scalar types: https://en.cppreference.com/w/cpp/language/type
> scalar types are (possibly cv-qualified) object types that are not array types or class types (see also std::is_scalar);

So should be nullptr without having to initialize it explicitly here.

Reply:
done